release.py

```diff
@@ -514,6 +519,8 @@ def main(argv):
     chdir_to_repo_root()
     parser = get_arg_parser()
     options, args = parser.parse_args(argv)
+    if options.skip_docs and not options.export:
+        error("--skip-docs option has no effect without --export")
```

Comment:
I was going to suggest using [`add_mutually_exclusive_group`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) instead, but then saw we're using the deprecated optparse rather than argparse!

Something for another time :)